inbc-program/inbc/parser.py

```diff
@@ -414,7 +414,7 @@ def pota(args) -> str:
         'biosversion': args.biosversion,
         'manufacturer': args.manufacturer,
         'product': args.product,
-        'release_date': args.releasedate,
+        'release_date': args.release_date,
         'Target': args.target,
```

Comment:
This is the actual fix.  It was using the FOTA release date instead of the SOTA one for SOTA.